docs/players.rst

```diff
@@ -95,7 +95,15 @@ The stable version of VLC cannot play Youtube Live streams. This a known bug
 that is fixed in VLC 3, which is still in development. You can also try
 using a different player.
 
+Youtube Live does not work with Mplayer
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Some versions of Mplayer cannot play Youtube Live streams. And errors like:
+Cannot seek backward in linear streams!
```

Comment:
```suggestion

.. code-block:: console

    Cannot seek backward in linear streams!
    Seek failed

```

If you set the error messages into a code-block for console output (mind the white space), it'll make it more distinguishable from the regular text.